test/poly1305_internal_test.c

```diff
@@ -37,7 +37,6 @@ typedef struct {
 
 static void benchmark_poly1305()
 {
-# ifdef OPENSSL_CPUID_OBJ
```

Comment:
Exposing this to anything else but real TSC makes no sense. And I think I already said this before, this is effectively debugging output, [customarily] one has to create condition for making it meaningful, and one even might have know how to interpret it. In other words it has *no* value to common user. With this in mind it actually makes more sense to remove it altogether.

Reply:
It seems like it would make sense for a nanosecond or microsecond resolution timer.
I agree that much slower than these and it is meaningless.

Reply:
Since you removed the `OPENSSL_CPUID_OBJ` guard, you need to check the return value of `OPENSSL_rdtsc` instead (line 54).